src/manipulators/manipulator.ts

```diff
@@ -54,7 +54,7 @@ export abstract class Manipulator {
    * @param {string} comment - Comment to look for when parsing leading comment ranges of node.
    * @return {boolean} True if node hasn't been editted before.
    */
-  verifyCommentRange(node: Node<ts.Node>, comment: string): boolean {
+  static verifyCommentRange(node: Node<ts.Node>, comment: string): boolean {
```

Comment:
We can put static function to a util class, not prefer to put in "abstract" class since they should be abstract.

Reply:
I feel that single monolithic util classes are a bit overused and can cause a large amount of confusion due to them turning into a catch all dumping ground for functions. I also don't see an issue with adding static functions to an Abstract class. Not all abstract classes should be purely abstract.

If the utility class is directly related to the functionality of the Abstract class I think it is ok to leave them in the context that makes sense.

In the case of things like map and list utility functions, which are not related to a specific class or concept you created, you can create specific utility classes (i.e. collection_util.ts). This will keep things more organized.

Just my 2 cents.

Reply:
For now, I'll move the map and list utility functions and move them to `collection_util,ts`. Another option is to create multiple util classes, for ex. `types_util.ts` that can contain those util functions instead of having all of the functions in a single util class. @jiady we can discuss the final implementation during our checkin today

Reply:
After discussing with Dongyu, we decided to create two util classes, one for collections (adding to mapsets, etc.) and one for manipulators.